src/view/03_components.md

```diff
@@ -429,6 +429,50 @@ view! {
 }
 ```
 
+``````admonish note
+If you would like to extract the attributes into the function so you can use it in multiple components, you can do so by implementing a function that returns `impl Attribute`.
+
+That would make example above look like this:
+
+```rust
+fn spread_onto_component() -> impl Attribute {
+    view!{
+        <{..} aria-label="a component with attribute spreading">
+    }
+}
+
+#[component]
```

Comment:
I don't think this needs to duplicate the entire component above: it might be easier to understand if you simply show the function that returns `-> impl Attribute`. I suppose if you think it's important to show how to use it in a component, it could simply be something like `<SomeComponent {..spread_onto_component()}/>`.